torch/nn/functional.py

```diff
@@ -1965,7 +1965,11 @@ def embedding_bag(input, weight, offsets=None, max_norm=None, norm_type=2,
                                   "(got mode='{}'). Please open a feature request on GitHub."
                                   .format(mode))
 
-    ret, _, _, _ = torch.embedding_bag(
+    op = torch.embedding_bag
```

Comment:
This seems pretty hacky. With my proposed structure above you'll have a single high level op which is still `torch.embedding_bag` and this part won't be necessary I think